webpack.common.js

```diff
@@ -205,7 +205,7 @@ const config = {
                     path.resolve(__dirname, 'node_modules/ssr-window'),
                     path.resolve(__dirname, 'node_modules/swiper'),
                     path.resolve(__dirname, 'node_modules/usehooks-ts'),
-                    path.resolve(__dirname, 'src')
+                    path.resolve(__dirname, 'node_modules/')
```

Comment:
```suggestion
                    path.resolve(__dirname, 'src')
```
and keep this list alphabetical.

But I would wait to see what Bill has to say.

Reply:
Woops sorr about that